### ISSUE TITLE
[Accessibility]: Stats and tooltips in repositories page are inaccessible

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#41624]
### Audit type

Screen reader navigation

### Problem description

Navigate to the repositories page under the site-admin settings. Visually, statistics for total number of repos + a breakdown for different states is visible. However, these numbers and the tooltips for the text are not accessible to a screen reader.

Tested with Firefox 104.0.2.

### Expected behavior

Statistics and tooltips should be accessible to screen readers.

### Additional details

See attached screen recording.

https://user-images.githubusercontent.com/93103176/189646361-75565a8c-921d-4515-aeb2-db1cfa88740a.mov

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by varungandhi-src** [comment]
There is a bunch of good discussion in [Tooltips in the time of WCAG 2.1](https://sarahmhigley.com/writing/tooltips-in-wcag-21/) on creating accessible tooltips.

**Comment by felixfbecker** [comment]
I think this is a global problem with our `<Tooltip>` component. Quoting my comment from https://github.com/sourcegraph/sourcegraph/issues/41683:

> Another problem here is that the tooltip is not announced. I think this is also because of the wrapper element, `aria-describedby` is on the parent element, not on the wrapper `div` that has `tabindex="0"`. I don't know how this one can be fixed considering that `<Tooltip>` is currently always a wrapper component. Maybe the right API for `<Tooltip>` would be that `children` is a render prop that passes a props object with `aria-describedby` to spread into the child.

**Comment by felixfbecker** [comment]
This actually appears to be fixed now. The issue above only applies to disabled buttons.
